src/TropicalGeometry/matrix.jl

```diff
@@ -41,3 +41,43 @@ end
 function det(A::Matrix{TropicalSemiringElem{minOrMax}}) where {minOrMax<:Union{typeof(min),typeof(max)}}
     return det(matrix(TropicalSemiring{minOrMax},A))
 end
+
+@doc raw"""
+    is_tropically_generic(A::MatrixElem{<:TropicalSemiringElem}, minOrMax)
+
+Check if a collection of vectors in the tropical torus (given as columns of a matrix `A`) are in tropical general position with respect to the `minOrMax` convention.
+
+# Examples
+```jldoctest
+julia> A = matrix(tropical_semiring(),[1 0;0 1])
+[(1)   (0)]
+[(0)   (1)]
+
+julia> is_tropically_generic(A,min)
+true
+```
+"""
+function is_tropically_generic(A::MatrixElem{<:TropicalSemiringElem}, minOrMax)
+    @req convention(A) == minOrMax "Semiring convention not as declared"
+    if ncols(A) == nrows(A)
+        return Polymake.tropical.tsgn(A) != 0
+    elseif ncols(A)>nrows(A)
+	i = 0
+        for b in subsets(ncols(A), nrows(A))
+            if Polymake.tropical.tsgn(A[:,b]) == 0
+                return false
+		break
+	    end
+        end
```

Comment:
The indentation looks a little bit off. Could it be that you mix both tabs and spaces? The Oscar style guide prefers spaces